lib/app_web/live/app_live.ex

```diff
@@ -98,17 +98,20 @@ defmodule AppWeb.AppLive do
     tag = Tag.get_tag!(value["tag_id"])
     tags = Tag.list_person_tags(person_id)
 
-    selected_tags =
-      if Enum.member?(selected_tags, tag) do
-        List.delete(selected_tags, tag)
-      else
-        [tag | selected_tags]
-      end
-      |> Enum.sort_by(& &1.text)
+    selected_tags = toggle_tag(selected_tags, tag)
 
     {:noreply, assign(socket, tags: tags, selected_tags: selected_tags)}
   end
 
+  defp toggle_tag(selected_tags, tag) do
```

Comment:
Can you place this function not in-between `handle_event` functions? So the file is correctly formatted, `mix format` is tossing this error:

```
clauses with the same name and arity (number of arguments) should be grouped together, "def handle_event/3" was previously defined (lib/app_web/live/app_live.ex:45)
```

This can easily be solved by just placing the private function alongside the others and let the `handle_event` functions be grouped together 😄

Reply:
You are right, sorry for that! it's fixed!